### DISCUSSION TITLE
Laravel - Active Route - Prefix Wildcard

### DISCUSSION THREAD

**Discussion by ekcltfrm** [tighten/ziggy#642] [Q&A] [answered]
```
 <NavLink
                                    href={route("dashboard")}
                                    active={route().current("dashboard")}
                                >
                                    Your Dashboard
                                </NavLink>
```

This works great for the page, however I've grouped routes with prefixes. I want to check it against the parent prefix of the route. How could I modify this?

Thank you.

**Comment by bakerkretzmar** [comment]
If you have route names like `dashboard.home`, `dashboard.profile`, `dashboard.billing` etc. you can call `route().current('dashboard.*')`. Is that what you mean?

Docs: https://github.com/tighten/ziggy#the-router-class

**Comment by ekcltfrm** [comment]
I've grouped routes like below with a prefix. 

```
Route::group(['prefix'=>'sl'], function () {
                Route::middleware('web', 'auth')
                ->prefix('sales')
                ->group(base_path('routes/salesroutes.php'));

                Route::middleware('web', 'auth')
                ->prefix('product')
                ->group(base_path('routes/productroutes.php'));
            });
```

I have a top nav bar. There is top a level item in the nav, I want it to be highlighted if current route has the "sl" prefix.  I would say there is about more than 50 routes. Routes are not really named in the way you mentioned. Is that the only way to do it?

**Reply by bakerkretzmar** [reply, marked as answer]
Yeah Ziggy's 'current' functionality only works with named routes. If your routes all start with `/sl/` but aren't named `sl.*` you don't really need Ziggy to make those checks, you could do something like `location.pathname.startsWith('/sl/')`.

**Reply by ekcltfrm** [reply]
thank you!
